src/main/java/reactor/ipc/netty/http/client/HttpClientOperations.java

```diff
@@ -114,6 +114,8 @@ static HttpOperations bindHttp(Channel channel,
 		this.redirectable = replaced.redirectable;
 		this.inboundPrefetch = replaced.inboundPrefetch;
 		this.requestHeaders = replaced.requestHeaders;
+		this.clientError = replaced.clientError;
```

Comment:
since there was a copy-constructor bug (partial copy), would it make sense to add a test case that checks the copy constructor?